man/nvim.1

```diff
@@ -0,0 +1,417 @@
+.Dd May 7, 2015
+.Dt NVIM 1
+.Os Neovim
+.Sh NAME
+.Nm nvim
+.Nd edit text
+.Sh SYNOPSIS
+.Nm
+.Op Ar options
+.Op Ar file ...
+.Nm
+.Op Ar options
+.Fl
+.Nm
+.Op Ar options
+.Fl t Ar tag
+.Nm
+.Op Ar options
+.Fl q Ar errorfile
+.Sh DESCRIPTION
+.Nm
+is a text editor based on
+.Xr vim 1 .
+To enter commands in
+.Nm ,
+type a colon
+.Qq ( Cm \&: ) ,
+which is also used in this manual to denote commands.
+For more information, consult the on-line help system with the
+.Ic :help
+command.
+.Bl -tag -width Fl
+.It Ar file ...
+File(s) to edit.
+If none are specified, open an empty buffer.
+If multiple files are specified, open one buffer for each file.
+To switch between buffers, use the
+.Ic :next
+and
+.Ic :previous
+commands.
+.It Fl
+Read text from standard input until EOF, then open a buffer with that text.
+Commands are read from standard error, which should be associated with a
+terminal.
```

Comment:
> I don't get the insistence on TTY being a misnomer. Literally everything in computing is an analogy of some physical concept. TTY is a technical term with a long history in unix, as is "socket", "port", "core dump", "page"

I hadn't thought about that, although It's still a misnomer in that it's usually not used in the context of Teleprinters, where are terminal makes more sense here because it's just a front-end to a system (like an interactive shell is to a login shell). It doesn't really matter much.

> It makes no sense to try to make it friendly, it won't help any user (who probably also doesn't know the difference between a terminal and a shell--not to mention a pseudo-terminal...).

I suppose I wasn't clear, but I'm more concerned about it being correct than friendly (not that they're mutually exclusive). And since "correct" isn't so easily defined in this context, I mean following the de facto standards regarding man pages.

The main point I'm making is that `tty` is to `terminal` as `stdXXX` is to `standard XXX`, at least from what I've gathered from various man pages. `tty` looks to be used almost exclusively in a non-general way, things like function names or `/dev` paths, which is no surprise given how UNIX influenced POSIX. However, even POSIX's [`man isatty`](http://pubs.opengroup.org/onlinepubs/009695399/functions/isatty.html) (among many others) uses the term "terminals", as there's no need to continue to use a misnomer just because the name is used. Most man pages I've read agree as well.